examples/phi3/vision/config_templates/vision_config.json

```diff
@@ -0,0 +1,46 @@
+{
+    "input_model": {
+        "type": "PyTorchModel",
+        "model_path": "<<str: model_path>>",
+        "model_loader": "vision_embed_tokens_loader",
+        "model_script": "scripts/user_script.py",
```

Comment:
this path is relative to the "vision" folder but we are running from "phi3"

Reply:
The value will be updated in phi3-vision.py. To avoid confusion, updated.